test/test_nn.py

```diff
@@ -7032,6 +7032,10 @@ def test_l1_loss_correct(self):
                 torch.nn.L1Loss()(input, torch.zeros_like(input)),
                 input.abs().mean())
 
+    def test_smoothl1loss_negative_beta_not_supported(self):
+        with self.assertRaises(RuntimeError):
```

Comment:
nit: you can use `assertRaisesRegex` to also verify that the error message is as expected.